telemetry/qml/TextLabel.qml

```diff
@@ -17,18 +17,42 @@
 //  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
 //=============================================================================
 
-import QtQuick 2.0
+import QtQuick 2.5
```

Comment:
Could even go up to `2.9` ;-)

Reply:
I never know though how to find out, except with trial and error how high one can get, and also not how high one needs to get

Reply:
There is [a table](https://doc.qt.io/qt-5/qtquickcontrols-index.html#versions) which could help to find this out. But since this versioning actually works only for object types and is broken for their interfaces (see QTBUGs [26888](https://bugreports.qt.io/browse/QTBUG-26888) and [30537](https://bugreports.qt.io/browse/QTBUG-30537)) there is no much point in trying to import the latest version as long as all the necessary types are available.

Reply:
Last activity on those Qt bugs 6 years ago :-(
But thanks for that table ;-)